met-api/src/met_api/models/submission.py

```diff
@@ -192,3 +201,24 @@ def get_engaged_users(cls, engagement_id) -> List[User]:
             .filter(Survey.engagement_id == engagement_id)\
             .all()
         return users
+
+    @staticmethod
+    def _filter_by_advanced_filters(query, advanced_search_filters: dict):
+        if status := advanced_search_filters.get('status'):
+            query = query.filter(Submission.comment_status_id == status)
+
+        if comment_date_to := advanced_search_filters.get('comment_date_to'):
+            query = query.filter(Submission.created_date <= comment_date_to)
+
+        if comment_date_from := advanced_search_filters.get('comment_date_from'):
+            query = query.filter(Submission.created_date >= comment_date_from)
+
+        if reviewer := advanced_search_filters.get('reviewer'):
+            query = query.filter(Submission.reviewed_by.ilike('%' + reviewer + '%'))
```

Comment:
not too sure ;but try using a f string 
query.filter(Submission.reviewed_by.ilike(f'%{reviewer}%'))

Reply:
done